### PR TITLE
[qa] Cleanup isort and flake8 settings in an OpenWISP module #30

### LINKED ISSUE #30: [qa] Cleanup isort and flake8 settings in an OpenWISP module

### LINKED ISSUE #32: [qa] Add commit message style check

### PR DESCRIPTION
**Pull request by ppabcd**
Clean isort, flake and add commit style check

Fixes #32 
Fixes #30

### CODE REVIEW

**Comment by ppabcd** [comment]
All done @nemesisdesign 
Only error from coverage. Flake updated.

File: runflake8 [hunk 1]
```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 set -e
-flake8 --max-line-length=110 \
-       --exclude=migrations,./tests/*settings*.py || exit 1
+flake8
```
Comment (nemesisdesign) [inline runflake8]: we can remove this file and call `flake8` directly in `.travis.yml`

File: runisort [hunk 2]
```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
-set -e
-isort --check-only --recursive --diff --skip migrations || exit 1
+isort --check-only --recursive --diff .
```
Comment (nemesisdesign) [inline runisort]: add the new line at the end of the file and configure your editor to always add it

File: requirements-test.txt [hunk 3]
```diff
@@ -3,3 +3,4 @@ coveralls
 isort
 flake8
```
Comment (nemesisdesign) [inline requirements-test.txt]: we should remove isort and flake8

Comment (ppabcd) [reply requirements-test.txt]: is isort and flake8 installed without requirement-test? @nemesisdesign

Comment (nemesisdesign) [reply requirements-test.txt]: like this here: https://github.com/openwisp/django-freeradius/pull/213
But wait for the build on that PR to complete so we can see if it really works :grin: (it should.. but we never know 100% sure until we try)

Comment (ppabcd) [reply requirements-test.txt]: ok i will waiting until build success and when all done i'll squash my commits.

**Comment by ppabcd** [comment]
@nemesisdesign  i think this errors because coverage.

**Comment by ppabcd** [comment]
@nemesisdesign I saw a previous build experiencing this problem. You can check here https://travis-ci.org/openwisp/openwisp-network-topology/builds/345852324

**Pull request merged**